CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.9.0 (2019-08-19)
+
+- SKY UX Builder will now reference a global instance of the `@blackbaud/auth-client` library for obtaining BBID JWTs and loading the omnibar when available. This allows the host page to add its own reference to `@blackbaud/auth-client`, enabling the library to become a "runtime" dependency that can be updated without redploying the SPA. If the host page does not provide a global instance of `@blackbaud/auth-client`, SKY UX Builder will fall back to its previous behavior of using the local version of the library included in the SPA's bundle. [#154](https://github.com/blackbaud/skyux-sdk-builder/pull/154)
```

Comment:
I'd recommend a few changes to tighten up and to follow the standard pattern for the first sentence of changelog entries. Here's a suggested rewrite with my changes implemented. (Let me know if you want me to explain the rationale for any of the changes.):

"Referenced a global instance of `@blackbaud/auth-client` to obtain BBID JWTs and load the omnibar. This enables the library to become a runtime dependency that you can update without redeploying the SPA. However, if the host page doesn't provide a global instance of `@blackbaud\auth-client`, then SKY UX Builder still uses the local version of the library in the SPA's bundle."

Reply:
Made suggested changes.